### ISSUE TITLE
not able to reproduce Chlorine results

### ISSUE THREAD

**Issue by mprostock** [NellyElsayed/GRU-FCN-model-for-univariate-time-series-classification#1]
Hi, thanks for publishing the code. Unfortunately I am not able to reproduce the stellar results on the chlorine dataset as shown in Table III of your paper (testing error of 0.002). The best accuracy I get is little over 80%, error of 0.19xxx. The only way I get 99-100% error is by reversing test and train sets.

Could you verify? It would also be helpful to freeze/list the library versions (keras etc.) that were used and publish a conda/pip env file, so that changes in underlying libs do not affect results.
Thanks.

**Comment by NellyElsayed** [comment]
Hello, Thanks for interesting in our work.  
About the result you received it is very weird because we recorded our results based on 3 different training processes for each dataset. Our paper has been accepted and soon I am going to upload all the trained weights (for the three trials of each dataset) so you can find the exact result as we have.
Just try to restart the kernel if you tried any other simulation before run the code. Might it will help. 

For the library that we used: Keras 2.0, Tensorflow 1.5, conda env, spyder, and Windows 10 OS.

**Comment by JakobSpahn** [comment]
I am also unable to reproduce your results. May I ask if you still plan on uploading the trained weights (sorry if late)?
